src/sql/workbench/services/objectExplorer/browser/objectExplorerService.ts

```diff
@@ -30,6 +31,25 @@ export interface NodeExpandInfoWithProviderId extends azdata.ObjectExplorerExpan
 	providerId: string;
 }
 
+export interface IServerTreeView {
```

Comment:
I need to abstract this so that the object explorer service wasn't directly refering to the tree from the workbench. This code should really be refactored to invert this dependency but thats future work.